workspace/utilities/admin-tools/admin-tools.xsl

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- 
+	Admin tools must be called in the context of the main entry of the page.
+	If called elsewhere, some widgets might break (edit-entry WILL break).
+-->
+
+<!-- IMPORTS -->
+<xsl:import href="widgets/widget-flush-cache.xsl" />
+<xsl:import href="widgets/widget-logout.xsl" />
+<xsl:import href="widgets/widget-edit-entry.xsl" />
+<xsl:import href="widgets/widget-debug.xsl" />
+
+	<xsl:template name="admin-tools">
```

Comment:
Do/should we require a comment before the template declaration ?

Reply:
no we don't

Reply:
@PascalPiche should we ?

Reply:
@PascalPiche Ping!
@nitriques like what ? How to use said template ?

Reply:
A little bit of doc wouldn't hurt anyone... 😬

Reply:
No I meant the comment used to make folding line easier

Reply:
No. We have a pretty good discipline right now keeping our templates small and tidy. If we need to fold code. Then the template is too big.

Reply:
Yeah. no need to add extra comments for folding.